app/gui/src/presenter/project.rs

```diff
@@ -303,14 +327,30 @@ impl Project {
         let weak = Rc::downgrade(&self.model);
         spawn_stream_handler(weak, notifications, |notification, model| {
             info!("Processing notification {notification:?}");
-            let message = match notification {
-                model::project::Notification::ConnectionLost(_) =>
-                    crate::BACKEND_DISCONNECTED_MESSAGE,
+            match notification {
+                model::project::Notification::ConnectionLost(_) => {
+                    let message = crate::BACKEND_DISCONNECTED_MESSAGE;
+                    let message = view::status_bar::event::Label::from(message);
+                    model.status_bar.add_event(message);
+                }
+                model::project::Notification::TextAutoSave => {
+                    model.check_project_changed();
+                }
```

Comment:
I would refactor it a bit. I think the logic of "check VCS status after autosave" should be covered in models/controllers, not in presenter. I prefer models: there could be additional notification (next to TextAutoSaved) about VCSStatus Change which is sent on each module change + the model::Project is the one calling check_vcs_status and sending a notification when it returns.

There should be a notification about "VCS status change", and here we just read it.